app/views/gaia.php

```diff
@@ -217,16 +218,21 @@
 
 $repo_one = $repos_nice_names[$englishchanges[0]];
 $repo_two = $repos_nice_names[$englishchanges[1]];
-$table = '<table id="englishchanges" class="collapsable">'
-       . '<tr>'
-       . '<th colspan="3">Strings that have changed significantly in English between ' . $repo_one . ' and ' . $repo_two . ' but for which the entity name didn\'t change</th>'
-       . '</tr>'
-       . '<tr>'
-       . '<th>Key</th>'
-       . '<th>' . $repo_one . '</th>'
-       . '<th>' . $repo_two . '</th>'
-       . '</tr>';
 
+$section_title = "Strings that have changed significantly in English between {$repo_one} and {$repo_two} but for which the entity name didn’t change";
+$table = "
+    <p class='section_title'>{$section_title}</p>
```

Comment:
or many a <th> that would span all columns if it's part of the table, or a <caption> tag?